src/routes/router.ts

```diff
@@ -18,14 +20,18 @@ import { createAppClient } from "~/src/util/get-github-client-config";
 import { GithubManifestGet } from "routes/github/manifest/github-manifest-get";
 import { GithubCreateBranchOptionsGet } from "~/src/routes/github/create-branch/github-create-branch-options-get";
 
+const logger = getLogger("router-config");
 export const RootRouter = Router();
 
 // The request handler must be the first middleware on the app
 RootRouter.use(Sentry.Handlers.requestHandler());
 
 // Parse URL-encoded bodies for Jira configuration requests
 RootRouter.use(urlencoded({ extended: false }));
-RootRouter.use(json());
+RootRouter.use(json({
+	limit: envVars.EXPRESS_PAYLOAD_LIMIT
+}));
```

Comment:
Why do we need this as an environment variable?  It's probably not ever going to change.  Plus, you're not using a default.

Probably don't need to make this bigger than 5mb

Reply:
The env is handy for unit testing of that settings is working. But If that's "over complicated", I can remove the unit test if you like.

For "50mb", I think it is safe to do so, as from github doc (in description of this pr), `25mb` is the max. So I just double it.

Reply:
...but why double it if it's the max?

Reply:
And if you want to test it, there are others ways of doing so without using an environment variable that'll never change.

Reply:
It can be 30mb, 35mb, or whatever number. I just don't want to sit on the edge of 25mb.

Reply:
Okay, let's recap on this:
1. I can remove it from env, I can agree the hardcoded value is simplier.
2. Do you think I should remove that test? Or just test against the hardcoded value (which can be done as well).
3. What number do you suggest? How about 30mb?

Reply:
I don't think you need to test it.  It's a library that we trust and I'd imagine they already have a plethora of tests for it already.  Plus we know it works as per the issue this morning :P

I really don't think we need to go anything above 25mb, even if it's on the edge, that's just the maximum github will ever send, but it doesn't come close to that 99.999999% of the time since the limit was just 100kb before.

Reply:
I agree with Michel; I don't think we should test the underlying libraries. Also don't think it should be an env variable, cause we have plenty of them TBH and each has its own cognitive toll.

Instead, I'd suggest to add a link to GitHub doc explaining why we picked 25mb, wdyt?

Reply:
Also yeah, sitting at 25mb might be dodgy (depending on how github calculates it). Should we have 20-25% of a gap just in case? Which is 31.25Mb.

Reply:
Okay thanks both @mboudreau and @bgvozdev. 
I've updated to just 30mb. I see no points arguing about 25mb vs 30mb since only 5mb different.
But if you guys are still thinking 1mb vs 10mb vs 25mb, then we can continuing talking about it.